pages/common/pabcnetcclear.md

```diff
@@ -0,0 +1,12 @@
+# pabcnetcclear
+
+> Preprocess and compile PascalABC.NET source files.
+> More information: <http://pascalabc.net/en/>.
+
+- Compile source file into executable:
```

Comment:
This is just a guess, but we should specify where the executable will be saved to.
```suggestion
- Compile a source file into an executable with the same name:
```